dags/bqetl_fxa_events.py

```diff
@@ -298,6 +298,18 @@
         depends_on_past=False,
     )
 
+    firefox_accounts_derived__fxa_users_services_first_seen__v1 = bigquery_etl_query(
+        task_id="firefox_accounts_derived__fxa_users_services_first_seen__v1",
+        destination_table="fxa_users_services_first_seen_v1",
+        dataset_id="firefox_accounts_derived",
+        project_id="moz-fx-data-shared-prod",
+        owner="dthorn@mozilla.com",
```

Comment:
@kik-kik you don't want to add yourself as an owner too?

Reply:
This value come from `metadata.yaml` in the dataset. I'll make an update there to make me the owner of this.